vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -963,6 +963,7 @@ class TimelineFragment @Inject constructor(
 
     override fun onDestroyView() {
         audioMessagePlaybackTracker.makeAllPlaybacksIdle()
+        audioMessagePlaybackTracker.clear()
```

Comment:
Can it be handled in `MessageComposerViewModel`, since we are calling `AudioMessageHelper.clearTracker()` there?

Reply:
Yes you are right. This is cleaner. I have used the `MessageComposerAction.EndAllVoiceActions` which in fact was not used. See [5c2e0ee](https://github.com/vector-im/element-android/pull/6679/commits/5c2e0ee7e61245fd1d627711ce05d83012f289a0).